sycl/include/sycl/handler.hpp

```diff
@@ -945,6 +946,34 @@ class __SYCL_EXPORT handler {
       sycl::ext::oneapi::experimental::execution_scope threadScope,
       sycl::ext::oneapi::experimental::execution_scope coordinationScope);
 
+  template <typename Properties>
+  void checkAndSetClusterRange(const Properties &Props) {
```

Comment:
Maybe adding:
```
namespace syclex = sycl::ext::oneapi::experimental
```
Within this func would make code more readable

Reply:
Fully qualified names should always be used within the compiler. This namespace definition can cause problems and would expose syclex to the user.

Reply:
That's not the case if you use the namespace only within a scope. Declaring this namespace in the func won't cause any namespace leaking.

Reply:
Done

Reply:
Thanks, good stuff.